website/docs/r/mssql_database.html.markdown

```diff
@@ -113,7 +113,7 @@ The following arguments are supported:
 
 * `short_term_retention_policy` - (Optional) A `short_term_retention_policy` block as defined below.
 
-* `sku_name` - (Optional) Specifies the name of the SKU used by the database. For example, `GP_S_Gen5_2`,`HS_Gen4_1`,`BC_Gen5_2`, `ElasticPool`, `Basic`,`S0`, `P2` ,`DW100c`, `DS100`. Changing this from the HyperScale service tier to another service tier will force a new resource to be created.
```

Comment:
This shouldn't be removed since there's a CustomizeDiff on the resource that applies.

Reply:
thanks! I missed this information, but I changed the description instead of recovering the old one. because my tool will detect the old one as a `ForceNew = true` property.